thirdeye-server/src/main/java/ai/startree/thirdeye/config/ThirdEyeServerConfiguration.java

```diff
@@ -68,8 +73,12 @@ public class ThirdEyeServerConfiguration extends Configuration {
   @JsonProperty("sentry")
   private BackendSentryConfiguration sentryConfiguration = new BackendSentryConfiguration();
 
+  @Deprecated // use defaultWorkspaceConfiguration instead
   @JsonProperty("time")
-  private TimeConfiguration timeConfiguration = new TimeConfiguration();
+  private TimeConfiguration timeConfiguration = null;
+
+  @JsonProperty("defaultWorkspaceConfiguration")
+  private NamespaceConfigurationDTO namespaceConfiguration = new NamespaceConfigurationDTO();
```

Comment:
shouldn't we have a config file here instead of directly using DTO object?
I know it will introduce one more level of mapping from config file to DTO

Reply:
yes in theory - it was a hack to get this done quickly.
the config should always be a perfect mapping of the DTO though. 
So not sure what's best. 
Introducing another level results in a lot of useless boilerplate.
But decoupling is still helpful at the configuration level for backward compatibility.

Reply:
as discussed we will proceed with the current implem and introduce a configuration level once the DTO has stabilized or if we need to decouple DTO and config.